server/api.go

```diff
@@ -665,6 +676,283 @@ func (a *API) getConnectedUsersFile(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (a *API) resetMocks(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we add system admin checks to these? I know that environment variable should only be set when testing, but just in case.

Reply:
Done